doc/user/installation.rst

```diff
@@ -39,7 +39,14 @@ Many of these programs are available in the package repositories of Linux
 distributions or in Homebrew_ for macOS.  Others can be installed with standard
 package managers such as Rubygems, NPM, Cabal, etc.
 
+.. important::
+
+   For a GUI Emacs on MacOS we recommend to install and configure
+   exec-path-from-shell_ to make Emacs use the proper ``$PATH`` and avoid a
```

Comment:
I find monospaced links somewhat strange to read, and in any case you can't easily monospace links in Rest.